internal/agent/agent.go

```
@@ -46,9 +50,36 @@ func RunAgent(agentCfg *config.AgentEnvs) error {
 	metricsRepo := repositories.NewAgentRepository(metricsStore, agentCfg.Log)
+
+	var conn *grpc.ClientConn
+	if agentCfg.GRPCAddress != "" {
+		var err error
+		conn, err = grpc.NewClient(agentCfg.GRPCAddress, grpc.WithTransportCredentials(insecure.NewCredentials()))
+		if err != nil {
+			agentCfg.Log.ErrorContext(ctx,
+				"Failed to create grpc connection",
+				helpers.ErrAttr(err),
+			)
+			return err
```

golangci-lint (1.62.2)

[error] error returned from external package is unwrapped: sig: func google.golang.org/grpc.NewClient(target string, opts ...google.golang.org/grpc.DialOption) (conn *google.golang.org/grpc.ClientConn, err error)

(wrapcheck)